packages/dds/sequence/api-report/sequence.legacy.public.api.md

```diff
@@ -0,0 +1,31 @@
+## Public API Report File for "@fluidframework/sequence"
```

Comment:
Are these all re-exports of public things from other packages?

Reply:
Never mind. This is the issue that arises when we re-export without leveraging `bundledPackages`. `sequence` should probably be made to bundle `merge-tree`, which would improve this situation.

Reply:
But as a sanity check: can you confirm that these do not appear in the actual rollup?